### ISSUE TITLE
Operator fails with "Failed to get watch namespace: WATCH_NAMESPACE must not be empty"

### ISSUE THREAD

**Issue by javajon** [RedisLabs/redis-enterprise-k8s-docs#171]
Start Kubernetes 1.18 instance on Katacoda, [for example a sandbox here](https://katacoda.com/javajon/courses/kubernetes-applications/nginx)

Following the install procedure at https://operatorhub.io/operator/redis-enterprise, steps to reproduce:

curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.1/install.sh | bash -s v0.18.1

kubectl create -f https://operatorhub.io/install/redis-enterprise.yaml

kubectl logs -n operators redis-enterprise-operator-xxxxx...
time="2021-05-10T21:57:28Z" level=info msg="Go Version: go1.10.3"
time="2021-05-10T21:57:28Z" level=info msg="Go OS/Arch: linux/amd64"
time="2021-05-10T21:57:28Z" level=info msg="operator-sdk Version: 0.0.5+git"
time="2021-05-10T21:57:28Z" level=fatal msg="Failed to get watch namespace: WATCH_NAMESPACE must not be empty"

**Comment by javajon** [comment]
I tripped on the rug at the entrance. Shouldn't the published and public instructions at both [operatorhub](https://operatorhub.io/operator/redis-enterprise) and [artifacthub](https://artifacthub.io/packages/olm/community-operators/redis-enterprise) be the latest? Perhaps, until it's ready, the redis-enterprise might be removed from those sites if the instructions are not valid? Perhaps the top of the pages should point us to the valid instructions?

**Comment by worldofgeese** [comment]
I agree with @javajon. If the operator is deprecated on OperatorHub, it should be removed. I'd prefer the operator be updated to work on OperatorHub.
